### PR TITLE
CHE-10590: Create image to build machine exec for terminal-extension and for task api extension.

### PR DESCRIPTION
**Pull request by AndrienkoAleksandr**
### What does this PR do?
Create image to build machine exec for terminal-extension and for task api extension.

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/10590

### CODE REVIEW

**Review by mmorhun — APPROVED** [review]

**Comment by sunix** [comment]
nice, few suggestions for README file:
- explain how to configure a workspace
- explain with a sample of code how a client would use it.
- add few architecture note, maybe a sequence diagram, requirements, and protocol used, etc ...

File: dockerfiles/machine-exec/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+ENV GOLANG_DOWNLOAD_URL https://golang.org/dl/go$GOLANG_VERSION.linux-amd64.tar.gz
+ENV GOLANG_DOWNLOAD_SHA256 b5a64335f1490277b585832d1f6c7f8c6c11206cba5cd3f771dcb87b98ad1a33
+# Go workspace path
+ENV GOPATH /go
+# Go installation path
+ENV PATH ${GOPATH}/bin:/usr/local/go/bin:${PATH}
+ENV MACHINE_EXEC_HASH b57b40eb2088271dc24f538f224f8c1ac474a777
+ENV MACHINE_EXEC github.com/eclipse/che-theia-terminal-plugin/machine-exec-server
+
+RUN yum -y update && yum install -y git && git --version && \
```
Comment (olexii4) [inline dockerfiles/machine-exec/Dockerfile:14]: " git --version "
Do we really need it?

**Review by olexii4 — APPROVED** [review]

File: dockerfiles/machine-exec/Dockerfile (hunk 1 above)
Comment (AndrienkoAleksandr) [reply dockerfiles/machine-exec/Dockerfile:14]: Nice catch, I will fix it.

**Comment by AndrienkoAleksandr** [comment]
Current pull request will be moved to the another repo (it will called che-machine-exec) https://github.com/eclipse/che/issues/10817

**Comment by AndrienkoAleksandr** [comment]
Close pull request on favor https://github.com/eclipse/che-theia-terminal-extension/pull/4 .  Repository https://github.com/eclipse/che-machine-exec contains own CI to build docker image.

**Pull request closed without merging**